tests/cephfs/clients/test_selinux_relabel.py

```diff
@@ -0,0 +1,409 @@
+import os
+import random
+import string
+import time
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
```

Comment:
Logs "http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TAUD4G/Validate_selinux_relabels_0.log" has a failure, please check
Logs "http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AKQ7T1/" seems for scenario2, please add scenario3 logs as per this code.

Reply:
> Logs "http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TAUD4G/Validate_selinux_relabels_0.log" has a failure, please check Logs "http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AKQ7T1/" seems for scenario2, please add scenario3 logs as per this code.

Yes. 1st log has failures because of the issue and the bz is updated in the code.
I realigned the scenarios later and updated the latest logs as a comment. 

I am sharing the recent logs again here : 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JOCVFL/